data/ggm0001/ggm001/ggm0001.ggm001.1st1K-grc1.xml

```diff
@@ -85,14 +85,14 @@
             <refsDecl n="CTS">
 
                 <cRefPattern n="section" matchPattern="(.+).(.+)"
```

Comment:
I think the @n here and in the next cRefPattern are switched around. I think this one should be "paragraph" and the second one should be "section: